src/Microsoft.Kusto.ServiceLayer/Microsoft.Kusto.ServiceLayer.csproj

```diff
@@ -33,6 +33,7 @@
     <PackageReference Include="Microsoft.Azure.OperationalInsights" />
     <PackageReference Include="Microsoft.SqlServer.DACFx" />
```

Comment:
>DACFx [](http://example.com/codeflow?start=51&length=5)

It wont matter much, but just use the same casing as the official name of the package: Microsoft.SqlServer.DacFx